dapr/proto/scheduler/scheduler_callback.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package dapr.proto.scheduler.v1;
+
+import "google/protobuf/empty.proto";
+
+service SchedulerCallback {
+    // Callback RPC for job schedule being at 'trigger' time
+    rpc TriggerJob(TriggerRequest) returns (google.protobuf.Empty) {}
```

Comment:
instead of empty, have a `TriggerResponse` message which is empty for extensibility.